lib/src/gregorian_calendar.dart

```diff
@@ -48,17 +48,16 @@ const List<int> _sakamotoHelper = const <int>[
 /// Represents a single day in the Gregorian Calendar
 class GregorianCalendar implements Calendar {
   /// Setting any of these to null treats them as 1 or 0 (year)
-  GregorianCalendar(int year, [int month = 1, int day = 1]) {
+  GregorianCalendar(this._year, [int? month = 1, int? day = 1]) {
```

Comment:
I think it'd be better to just make these non-null.

Reply:
I agree, for this constructor despite the comment on line 50 `Setting any of these to null treats them as 1 or 0 (year)` you should assume these are non-null.